integrations/clickup/pyproject.toml

```diff
@@ -0,0 +1,102 @@
+[tool.poetry]
+name = "clickup"
+version = "0.1.0"
```

Comment:
```suggestion
version = "0.1.0-beta"
```
for the integration to be considered as beta, it should firstly be released with a beta tag